events_structs.go

```diff
@@ -855,3 +858,69 @@ func (ev *EventStackChangedType) String() string {
 	d := ev.TypeData
 	return fmt.Sprintf("Stack changed from '%s' to %s", d.PreviousStackName, d.CurrentStackName)
 }
+
+// Database maintenance planned
+type EventPlanDatabaseMaintenancedTypeData struct {
+	AddonName     string `json:"addon_name"`
+	MaintenanceID string `json:"maintenance_id"`
```

Comment:
Since the [last update of the API](https://github.com/Scalingo/api/pull/2248) we probably want to add all available information,
i.e. adding:
- `maintenance_type`
- `maintenance_duration`
- `maintenance_duration`